client/app/scripts/liveblog-syndication/views/attach-syndicated-blogs-modal.html

```diff
@@ -23,7 +23,10 @@ <h3 translate>Attach Syndicated blog</h3>
                 <i class="icon-refresh"></i>
             </button>
         </div>
-        <ul class="producers-list">
+        <div ng-if="producerBlogs._items.length == 0" translate>
```

Comment:
I was wondering why this was passing jshint, then I realized that is in template.
Please use ===